### ISSUE TITLE
Show Only Expired Opportunities When You click for that.

### ISSUE THREAD

**Issue by 0himanshu3** [swapagarwal/swag-for-dev#1277]
## Issue regarding the website

- [ ] I've checked that this isn't a duplicate issue.

As i have seen as we click on show expired opportunities all opportunities are being showed.For a user if he wants something he would be intereseted in non expired and for exploring he would wish to see expired without every time looking for the tag.

So on click show expired opportunities i guess it would be better to show only expired ones.

If you also share sam thought please assign this issue to me under Hacktoberfest tag.

**Comment by Sadman95** [comment]
Hello! 
I think this is working except selecting Newest First in the Order selection. So when the newest opportunities load, there is no item tagged as expired.

**Comment by aslafy-z** [comment]
I think we can do two things:
- rename the current "Show expired opportunities" to "Include expired opportunities".
- make expired opportunities more obvious, maybe add a grey overlay?

Do we really need a "Show only expired opportunities" checkbox too?

**Comment by Sadman95** [comment]
I think it's more convenient to show expired before the others

**Comment by Tejas-Sharma-dev** [comment]
> I think we can do two things:
> 
> * rename the current "Show expired opportunities" to "Include expired opportunities".
> * make expired opportunities more obvious, maybe add a grey overlay?
> 
> Do we really need a "Show only expired opportunities" checkbox too?

I strongly agree, with this Displaying the new swag opportunity first, then proceeding to display expired with grey styling.

**Comment by 0himanshu3** [comment]
@aslafy-z 
 yes  we should change the name to "include expired opportunities" and add a gray layout for simplified view
 And checkbox is neccessary as some users only wish to see only the non-expired opporunties.
 
 If you agree with my though then please assign this task to me so i can modify it
